DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -98,9 +119,15 @@ function Set-TargetResource
                 if ($Version) {
                     Write-Verbose -Message "Uninstalling $Name due to version mis-match"
                     UninstallPackage -pName $Name -pParams $Params
-                    Write-Verbose -Message "Re-Installing $Name with correct version $version"
-                    InstallPackage -pName $Name -pParams $Params -pVersion $Version -pSource $Source -cParams $chocoParams
-                } elseif ($AutoUpgrade) {
+                    Write-Verbose -Message "Re-Installing $Name with correct version $versionToInstall"
+                    InstallPackage -pName $Name -pParams $Params -pVersion $versionToInstall -pSource $Source -cParams $chocoParams
+                }
+                elseif ($MinimumVersion) {
+                    Write-Verbose -Message "Upgrading $Name becuase installed version is lower that the specified minimum"
+                    $chocoParams += " --version '$versionToInstall'"

```

Comment:
When using Chocolatey parameters with `--` can you add an equals after it - `--version='$versionToInstall'`?